utils.py

```diff
@@ -48,6 +56,10 @@ def visualize_proba(proba, shape, color=(0, 0, 255)):
     return img
 
 
+def visualize_mask(image, mask):
```

Comment:
@Foxiy I believe the name of this function is off as it does not visualize anything as much as I can see.